### ISSUE TITLE
Category display and creation issue

### ISSUE THREAD

**Issue by agramian** [msjit/testcasedb#72]
There is some issue with the newest revision when creating and displaying categories.
Rails v3.2.21, Ruby v2.0
Please see the gif below.
![category-issue](https://cloud.githubusercontent.com/assets/7113257/6072833/4627b52a-ad5b-11e4-8d30-caf6a6b4aa6d.gif)

**Comment by agramian** [comment]
@joffems please take a look at this issue. It is completely breaking the ability to navigate around.  The database storage of the items seems correct with just one of each item stored, however duplicates are displayed.  I've attached a larger gif.

![20150213-171619_capture](https://cloud.githubusercontent.com/assets/7113257/6197966/53c2fa7e-b3a4-11e4-9f00-49a7925f0b7c.gif)

**Comment by kirantpatil** [comment]
I am at latest master branch and I added new product and created category and I don't see any issue.

kiran@jaihanuman:~/testcasedb$ git log
commit fc0218628119249f5b4c45d3147ace632d3770f7
Author: Marc Joffe <mjoffe@msjit.ca>
Date:   Sat Feb 7 00:23:26 2015 -0500

    Add basic Redis instructions to the README

![screenshot from 2015-02-14 16 53 48](https://cloud.githubusercontent.com/assets/1178055/6199447/1987a096-b46a-11e4-9762-108210c61266.png)

**Comment by agramian** [comment]
My Rails server is running in development mode on a Mac with OSX 10.9
I'll do a diff with my local code and the master branch and maybe try reinstalling TestCaseDB.

**Comment by joffems** [comment]
@agramian Are you still having this issue? I cannot reproduce this locally.

If it is still occurring, which browser are you using.

**Comment by agramian** [comment]
@joffems yes I still have this issue.  I am using the latest version of master with ruby-2.0.0-p598, Rails v3.2.21, Mac with OSX 10.9, Chrome Version 41.0.2272.89 (64-bit)
All objects are displayed as duplicates, not just the categories.  Are you using a mac to try and reproduce this?
